src/aihwkit/nn/modules/conv.py

```diff
@@ -167,3 +168,176 @@ def extra_repr(self) -> str:
+
+    Applies a 3D convolution over an input signal composed of several input
+    planes, using an analog tile for its forward, backward and update passes.
+
+    Note:
+        The tensor parameters of this layer (``.weight`` and ``.bias``) are not
+        guaranteed to contain the same values as the internal weights and biases
+        stored in the analog tile. Please use ``set_weights`` and
+        ``get_weights`` when attempting to read or modify the weight/bias. This
+        read/write process can simulate the (noisy and inexact) analog writing
+        and reading of the resistive elements.
+
+    Args:
+        in_channels: number of channels in the input image.
+        out_channels: number of channels produced by the convolution.
+        kernel_size: size of the convolving kernel.
+        stride: stride of the convolution-
+        padding: zero-padding added to both sides of the input.
+        dilation: spacing between kernel elements.
+        groups: number of blocked connections from input channels to output
+            channels.
+        bias: whether to use a bias row on the analog tile or not
+        padding_mode: padding strategy. Only ``'zeros'`` is supported.
+        rpu_config: resistive processing unit configuration.
+        realistic_read_write: whether to enable realistic read/write
+           for setting initial weights and read out of weights
+    """
+    # pylint: disable=abstract-method
+
+    __constants__ = ['stride', 'padding', 'dilation', 'groups',
+                     'padding_mode', 'output_padding', 'in_channels',
+                     'out_channels', 'kernel_size']
+    in_channels: int
+    out_channels: int
+    kernel_size: Tuple[int, int, int]
+    stride: Tuple[int, int, int]
+    padding: Tuple[int, int, int]
+    dilation: Tuple[int, int, int]
+    realistic_read_write: bool
+    fold_indices: Tensor
+    input_size: float
+    in_features: int
+    out_features: int
+
+    def __init__(
+            self,
+            in_channels: int,
+            out_channels: int,
+            kernel_size: Union[int, Tuple],
+            stride: Union[int, Tuple] = 1,
+            padding: Union[int, Tuple] = 0,
+            dilation: Union[int, Tuple] = 1,
+            groups: int = 1,
+            bias: bool = True,
+            padding_mode: str = 'zeros',
+            rpu_config: Optional[RPUConfigAlias] = None,
+            realistic_read_write: bool = False
+    ):
+        # pylint: disable=too-many-arguments
+        if groups != 1:
+            raise ValueError('Only one group is supported')
+        if padding_mode != 'zeros':
+            raise ValueError('Only "zeros" padding mode is supported')
+        if dilation != 1:
+            raise ValueError('Only dilation = 1 is supported')
+
+        kernel_size = _triple(kernel_size)
+        self.in_features = (in_channels // groups) * kernel_size[0] * kernel_size[1] * \
+            kernel_size[2]
```

Comment:
```suggestion
        self.in_features = ((in_channels // groups) *
                            kernel_size[0] * kernel_size[1] * kernel_size[2])  # type: ignore
```

Suggested change as `mypy` seems not to be smart enough to discover that after line 241, `kernel_size` will always be a 3-item tuple and no longer an `int`.